recipes/qtxlsxwriter/all/conanfile.py

```diff
@@ -0,0 +1,85 @@
+import os
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain
+from conans import ConanFile, tools
+
+
+class QtXlsxWriterConan(ConanFile):
+    name = "qtxlsxwriter"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/dbzhang800/QtXlsxWriter"
+    description = ".xlsx file reader and writer for Qt5"
+    topics = ("qtxlsxwriter", "excel", "xlsx", "conan-recipe")
+
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False]
+    }
+    default_options = {
+        "shared": False,
+        "fPIC": True
+    }
+
+    generators = "CMakeDeps"
+    exports_sources = [
+        "CMakeLists.txt",
+        os.path.join("patches", "*")
+    ]
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.configure()
+        return self._cmake
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        if self.options.shared:
+            del self.options.fPIC
+
+    def requirements(self):
+        self.requires("qt/5.15.2")
+
+    def generate(self):
+        tc = CMakeToolchain(self)
```

Comment:
🟨 For the Conan team, this is new to me, is it acceptable?

Reply:
yes, same thing. Too experimental to have it in conan center at the moment